src/core/components/NavMenu.tsx

```diff
@@ -2,6 +2,13 @@ import { Link, useLocation } from 'react-router-dom';
 import { useAppDispatch } from '../../config/redux/hooks';
 import { clearUserData } from '../../user/redux/userSlice';
 import { routes } from '../routing/routes';
+import { CustomButton } from '../components/buttons/Button/Button';
+import HomeIcon from '@mui/icons-material/Home';
+import CalendarMonthIcon from '@mui/icons-material/CalendarMonth';
+import LogoutIcon from '@mui/icons-material/Logout';
+import { useState, MouseEvent } from 'react';
+import { PendingActions } from '@mui/icons-material';
+import './NavMenu.css';
```

Comment:
Please make a new folder for the NavMenu component and move there the both files (.tsx and css). Please also add the index.ts and make a named re-export for the component, that should simplify the import

Reply:
Done!